pgsmo/objects/functions/templates_procedures/pg/14.0_plus/create.sql

```diff
@@ -0,0 +1,67 @@
+{#
+ # pgAdmin 4 - PostgreSQL Tools
+ #
+ # Copyright (C) 2013 - 2017, The pgAdmin Development Team
+ # This software is released under the PostgreSQL Licence
+ #}
+{% import 'security.macros' as SECLABEL %}
+{% import 'privilege.macros' as PRIVILEGE %}
+{% import 'variable.macros' as VARIABLE %}
+{% set is_columns = [] %}
+{% set exclude_quoting = ['search_path'] %}
+{% if data %}
+{% if query_for == 'sql_panel' and func_def is defined %}
+CREATE OR REPLACE PROCEDURE {{func_def}}
+{% else %}
+CREATE{% if add_replace_clause %} OR REPLACE{% endif %} PROCEDURE {{ conn|qtIdent(data.pronamespace, data.name) }}{% if data.arguments is defined %}
```

Comment:
For all the 
conn|qtIdent(data.pronamespace, data.name), we need to make 
conn|qtIdent(data.pronamespace, data.name)|replace('"', '')  to remove the quotation marks